spec/priorityQueue.js

```diff
@@ -0,0 +1,122 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import PriorityQueue from '../src/priorityQueue'
+
+chai.use(chaiChange)
+
+describe('PriorityQueue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(PriorityQueue).to.be.a('function')
+  })
+
+  context('enqueue()', () => {
```

Comment:
`enqueue` should also test that the correct *value* was inserted into the queue.